.sphinx/_static/furo_colors.css

```diff
@@ -85,3 +86,12 @@ body {
         }
     }
 }
+
+/* suppress link underlines outside on-hover */
+a {
+    text-decoration: none;
+}
+
+a:hover, a:visited:hover {
+    text-decoration: underline;
+}
```

Comment:
Oh, good that I commented on the colour definition - this part here should actually go into `custom.css`. I didn't notice in my first review, sorry!

The idea is that `furo_colors.css` has only the colour definitions (what you would usually do in the [`html_theme_options`](https://pradyunsg.me/furo/customisation/colors/), but having them in there makes our output files much bigger than needed), while all other custom CSS goes into the `custom.css` file.